### DISCUSSION TITLE
Is there an automatic way to run a 5-fold cross validation

### DISCUSSION THREAD

**Discussion by mchlsdrv** [MIC-DKFZ/nnUNet#526] [Q&A]
Hi, I'd want to use separate set of images for the test each fold, and I see that there is a Ts folder for the test images, and now what I do is place the images to be held-out of the training there each time, and train the nnUNet for 40 iterations each time. 
My question is, is there a way to specify the type of the cross-validation you want, e.g., 5-cross validation, and that the procedure will take care of the permutations?

Thanks in advance.
Michael

**Comment by FabianIsensee** [comment]
Hi,
test images belong in imagesTs. They will not be touched by nnU-Net unless you explicitly ask it to with nnUNet_predict.
The rest of the images belong in imagesTr. They will be split 80:20 for each fold (standard 5 fold cv). If you want a custom split, follow the guide https://github.com/MIC-DKFZ/nnUNet/blob/master/documentation/common_questions.md#creating-and-managing-data-splits
Best,
Fabian
